packages/components/src/components/page-error/page-error.jsx

```diff
@@ -49,31 +52,66 @@ const PageError = ({
                         'dc-page-error__message-wrapper--left': !!image_url,
                     })}
                 >
-                    <span
+                    <Text
+                        color='prominent'
                         className={classNames('dc-page-error__message', {
                             'dc-page-error__message--left': !!image_url,
                         })}
                     >
                         {messages.map((message, index) =>
                             message.has_html ? (
-                                <p
+                                <Text
+                                    as='p'
+                                    size='s'
+                                    lineHeight='x'
                                     key={index}
                                     className='dc-page-error__message-paragraph'
                                     dangerouslySetInnerHTML={{ __html: message.message }}
                                 />
                             ) : (
-                                <p key={index} className='dc-page-error__message-paragraph'>
+                                <Text
+                                    as='p'
+                                    size='s'
+                                    lineHeight='x'
+                                    key={index}
+                                    className='dc-page-error__message-paragraph'
+                                >
                                     {message}
-                                </p>
+                                </Text>
                             )
                         )}
-                    </span>
+                    </Text>
+                </div>
+                <div className='dc-page-error__btn-wrapper'>
+                    {redirect_urls ? (
```

Comment:
Should we do a `typeof` `array` check or `Array.isArray()` check before mapping?

Reply:
Now, the prop will only expect datatype of array, so we don't need to check